README.md

```diff
@@ -16,23 +18,26 @@
 [Slack](http://bit.ly/src-d-community) •
 [Twitter](https://twitter.com/sourcedtech)
 
-
 ## Introduction
 
-source{d} Engine exposes powerful Universal AST's to analyze your code and a SQL engine to analyze your git history:
+The source{d} Engine exposes powerful [Universal ASTs](#babelfish-uast) to analyze your code and a SQL engine to analyze your git history:
 
 - **Code Retrieval**: retrieve and store git repositories as a dataset.
-- **Language Agnostic Code Analysis**: automatically identify languages, parse source code, and extract the pieces that matter in a completely language-agnostic way.
+- **Language-Agnostic Code Analysis**: automatically identify languages, parse source code, and extract the pieces that matter in a completely language-agnostic way.
```

Comment:
Some of this was pre-existing, but the repetition of "language-agnostic" seems unnecessary. I suggest, maybe, "Automatically identify languages, parse source code, and extract the pieces that matter with language-independent queries." or words to that effect.